fe/fe-core/src/main/java/com/starrocks/connector/RemoteFileDesc.java

```diff
@@ -121,6 +123,15 @@ public String getFullPath() {
         return this.fullPath;
     }
 
+    public RemoteFileDesc setOrcFormatDesc(ORCFormatDesc orcFormatDesc) {
```

Comment:
it's best to use session variables. We need additional architecture to support this kind properties in catalog. The current change is not smooth.